flow/scripts/global_place.tcl

```diff
@@ -66,5 +66,4 @@ report_metrics "global place" false
 
 if {![info exists standalone] || $standalone} {
   write_def $::env(RESULTS_DIR)/3_1_place_gp.def
```

Comment:
my preference would be to treat the file writing as checkpoints (which were supposed to change to db files) and always write them, so the if {![info exists standalone] || $standalone goes away.

Reply:
I'm still deep in a chain of issues on using db files (write_def sorts the nets & insts; write_db doesn't which turns up problems).

I would have a check point control that is on by default.  That way someone who wants to save disk space has the option to disable it.

Reply:
@jjcherry56 and @maliberty I think changing checkpoints falls out of the scope of this PR. I will create a Jira to discuss and later address in a different PR.

Reply:
fine